photon-core/src/main/java/org/photonvision/vision/camera/LibcameraGpuSettables.java

```diff
@@ -114,20 +117,41 @@ public double getFOV() {
 
     @Override
     public void setAutoExposure(boolean cameraAutoExposure) {
-        lastExposureMode = cameraAutoExposure;
-        // TODO (Matt) -- call LibCameraJNI's auto exposure function, when that exists
-        LibCameraJNI.setAutoExposure(cameraAutoExposure);
+        lastAutoExposureActive = cameraAutoExposure;
+
+        // HACK!
+        // OV 9281 does not currently support autoexposure in our driver
+        // Just skip setting it silently
+        if (sensorModel == LibCameraJNI.SensorModel.OV9281) {
+            lastAutoExposureActive = false;
```

Comment:
Does it still not?

Reply:
Yea I'm still having issues with https://github.com/PhotonVision/photon-libcamera-gl-driver/suites/10144555465/artifacts/495489276

Reply:
@gerth2 we good now?

Reply:
yup all good